<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectrumEditorCustomDelta</class>
 <widget class="QWidget" name="SpectrumEditorCustomDelta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelFunctionExp">
     <property name="text">
      <string>Delta function</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>A:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineAmplitude"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>x0:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineStart"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>w:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineWidth"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
